matrix/system_of_linear_equation_in_2_variables.py

```diff
@@ -0,0 +1,84 @@
+def calculate_x_and_y(eq1, eq2):
+    """
+    Solves the system of linear equation in 2 variables.
+    :param: eq1: list of 3 numbers
+    :param: eq2: list of 3 numbers
+    :return: String of result
+    Theory:-
+    https://www.mathsisfun.com/algebra/systems-linear-equations-matrices.html
+    Cramer's rule for 2x2 matrix:-
+    https://www.chilimath.com/lessons/advanced-algebra/cramers-rule-with-two-variables
+    a1x + b1y = = d1
+    a2x + b2y = = d2
```

Comment:
Please could you move this to a docstring at the top of the file

Reply:
sure sir/mam

Reply:
Do you mean outside the function and at the top? right?